Detectors/MUON/MCH/DevIO/Digits/digits-writer-workflow.cxx

```diff
@@ -142,6 +146,7 @@ void customize(std::vector<ConfigParamSpec>& workflowOptions)
                                ConfigParamSpec::HelpString{"do not expect, in addition to digits and rofs, to get Orbits at the input"});
   workflowOptions.emplace_back(ConfigParamSpec{"input-digits-data-description", VariantType::String, "DIGITS", {"description string for the input digits data"}});
   workflowOptions.emplace_back(ConfigParamSpec{"input-digitrofs-data-description", VariantType::String, "DIGITROFS", {"description string for the input digit rofs data"}});
+  workflowOptions.emplace_back(ConfigParamSpec{"enable-root-output", VariantType::Bool, false, {"output in Root format"}});
```

Comment:
So far all workflows have global option `--disable-root-output`, which, if activated, simply does not attach the corresponding writer device. Can we have the same approach for the MCH, to be able to use common settings?

Reply:
well, this `digits-writer` device has been used for debug mainly, with a binary (non Root) format (this PR only adds the ability to use the Root format) . It's not supposed to be used "routinely" I would say.